### PR TITLE
Made some changes to the Introduction

### PR DESCRIPTION
**Pull request by vladimirias**
I have corrected some places on the first two or three pages of the introduction. Hopefully I removed some things that were wrong without adding new things that are wrong...

### CODE REVIEW

File: introduction.tex [hunk 1]
```diff
@@ -33,12 +33,17 @@ \chapter*{Introduction}
...
 
-Type theory was originally invented by Bertrand Russell \cite{Russell:1908},\index{Russell, Bertrand} as a device for blocking the paradoxes in the logical foundations of mathematics  that were under investigation at the time. It was later developed as a rigorous formal system in its own right (under the name ``$\lambda$-calculus'') by Alonzo Church \cite{Church:1933cl,Church:1940tu,Church:1941tc}.  Although it is not generally regarded as the foundation for classical mathematics, set theory being more customary, type theory still has numerous applications, especially in computer science and the theory of programming languages~\cite{Pierce-TAPL}.
+The concept of a type as the domain of significance of a propositional function was introduced by Bertrand Russell \cite{Russell:1908},\index{Russell, Bertrand} as a device for blocking the paradoxes in the logical foundations of mathematics  that were under investigation at the time. The next milestone in the development of type theory is the paper by Alonzo Church \cite{Church:1940tu} that connected types with the formal system called $\lambda$-calculus  that Church has developed for dealing with functions.
```
Comment (pthariensflame) [inline introduction.tex]: The end of this paragraph, with "Church has developed," could do with some grammatical correction.

**Comment by awodey** [comment]
Independently of whether these particular revisions are improvements, there is the question of whether we are permitting this degree of rewriting of the book.  We have not previously, and I don’t think we should start now — much of the book, and the introduction in particular, is the result of a process of careful collaboration with several rounds of revision that people eventually signed off on.  We can’t just rewrite whole passages like this now.  I think we should be only doing small corrections of material errors.

There is a possibility of a second edition at some point, and that would be an opportunity for major rewriting.

**Comment by mikeshulman** [comment]
I agree with @awodey.  For example, I wanted to make some other changes to the intro in issue #763, but eventually decided that would be too much change for an update to the 1st edition.

We do want to _correct_ anything that is _wrong_, but I don't see anything that is actually incorrect about the passages you've suggested to remove.

**Comment by vladimirias** [comment]
This

> Type theory .... was later developed as a rigorous formal system in its own right (under the name “λ-calculus”) by Alonzo Church [Chu33, Chu40, Chu41].

is a material error.

The paper Chu33 does not discuss type theory. In fact it is the second in a series of two papers where Church proposes an alternative to type theory.

Lambda-calculus is not a type theory.

Vladimir.

**Comment by mikeshulman** [comment]
What's a fix requiring minimal changes? Add the word "typed" in front of
"λ-calculus" and give the correct citations?

On Feb 28, 2015 10:13 AM, "Vladimir Voevodsky" <notifications@github.com>
wrote:
>
> This
>
> > Type theory .... was later developed as a rigorous formal system in its
own right (under the name “λ-calculus”) by Alonzo Church [Chu33, Chu40,
Chu41].
>
> is a material error.
>
> The paper Chu33 does not discuss type theory. In fact it is the second in
a series of two papers where Church proposes an alternative to type theory.
>
> Lambda-calculus is not a type theory.
>
> Vladimir.
>
>
>
>
>
>
>
> > On Feb 27, 2015, at 10:12 AM, Steve Awodey <notifications@github.com>
wrote:
> >
> > Independently of whether these particular revisions are improvements,
there is the question of whether we are permitting this degree of rewriting
of the book. We have not previously, and I don’t think we should start now
— much of the book, and the introduction in particular, is the result of a
process of careful collaboration with several rounds of revision that
people eventually signed off on. We can’t just rewrite whole passages like
this now. I think we should be only doing small corrections of material
errors.
> >
> > There is a possibility of a second edition at some point, and that
would be an opportunity for major rewriting.
> >
> >
> >
> > > On Feb 26, 2015, at 3:18 PM, Vladimir Voevodsky <
notifications@github.com> wrote:
> > >
> > > I have corrected some places on the first two or three pages of the
introduction. Hopefully I removed some things that were wrong without
adding new things that are wrong...
> > >
> > > You can view, comment on, or merge this pull request online at:
> > >
> > > https://github.com/HoTT/book/pull/773 <
https://github.com/HoTT/book/pull/773>
>
> > > Commit Summary
> > >
> > > Made some changes to the Introduction
> > > File Changes
> > >
> > > M introduction.tex <https://github.com/HoTT/book/pull/773/files#diff-0>
(28)
> > > Patch Links:
> > >
> > > https://github.com/HoTT/book/pull/773.patch <
https://github.com/HoTT/book/pull/773.patch>
> > > https://github.com/HoTT/book/pull/773.diff <
https://github.com/HoTT/book/pull/773.diff>
> > > —
> > > Reply to this email directly or view it on GitHub <
https://github.com/HoTT/book/pull/773>.

**Comment by vladimirias** [comment]
This

> The idea of homotopy type theory arose around 2006 in independent work by Awodey and Warren [AW09] and Voevodsky [Voe06], but it was inspired by Hofmann and Streicher’s earlier groupoid interpretation [HS98].
> 

is material error. The idea of generalizing the groupoid interpretation of Hofmann and Streicher to an interpretation in Kan complexes or weak factorization systems or Quillen model categories was widely discussed in the Fall of 2006. Among those discussing it there were Streicher himself, Richard Garner and Michael Warren. The notes written at about that time by Garner and Warren are available online.

Vladimir.






> 
>

**Comment by vladimirias** [comment]
Church was building on the work of others - in fact the title of the relevant paper (Chu40) is

“A formulation of *the* simple theory of types”

and he refers to the works where “the simple theory of types” for which he now gives a simple formulation
was constructed.

What is today called “typed lambda-calculus” is not what Church described in Chu40. What he described there is now called “higher-order logic”.

Replacing the sentence

"It was later developed as a rigorous formal system in its
own right (under the name “λ-calculus”) by Alonzo Church [Chu33, Chu40,
Chu41]."

with the sentence:

"This was followed by a period of development culminating with the paper by Alonzo Church \cite{Church:1940tu} where Church connected type theory with $\lambda$-calculus.”

is the smallest change that would fix that I can think of.

Vladimir.

**Comment by mikeshulman** [comment]
That replacement sentence sounds fine to me, but I'm not an expert in the
history. Anyone who is want to concur (or disagree)?

**Comment by awodey** [comment]
Church’s system was a formulation of an idea that had been developing for some time.
Between Russell and Church were Chwistek, Wittgenstein, Ramsey, Carnap, Gödel, Hilbert-Ackermann, Bernays, and others.
The use of a variable-binding operation for funciton abstraction goes back to Frege (who took it from integration).
Church introduced the \lambda notation and he based his system on the systematic use of it, first in the untyped form (as Frege had done), then typed as in Russell and others, most notably Gödel, whose 1931 paper contains a formulation of simple type theory.

The term “higher-order logic” is more recent, and is used to distinguish this system from “first-order logic”, the emphasis on which is a more modern development.  Hilbert-Ackerman and later Hilbert-Bernays introduced the distinction between “function calculus of first- and higher-order”.  Up to notational translation, there is no real difference between what is now called higher-order logic and simple type theory with ground types of individuals and propositions, as in Church’s 1941 system. 

None of this belongs in the book.
A small revision of the sentence in question might be:

> "It was eventually developed into a rigorous formal system in its own right under the name “typed λ-calculus”, see Church [Chu40, Chu41] for an early formulation."



Steve

**Comment by awodey** [comment]
Church’s system was a formulation of an idea that had been developing for some time.
Between Russell and Church were Chwistek, Wittgenstein, Ramsey, Carnap, Gödel, Hilbert-Ackermann, Bernays, and others.
The use of a variable-binding operation for funciton abstraction goes back to Frege (who took it from integration).
Church introduced the \lambda notation and he based his system on the systematic use of it, first in the untyped form (as Frege had done), then typed as in Russell and others, most notably Gödel, whose 1931 paper contains a formulation of simple type theory.

The term “higher-order logic” is more recent, and is used to distinguish this system from “first-order logic”, the emphasis on which is a more modern development.  Hilbert-Ackerman and later Hilbert-Bernays introduced the distinction between “function calculus of first- and higher-order”.  Up to notational translation, there is no real difference between what is now called higher-order logic and simple type theory with ground types of individuals and propositions, as in Church’s 1941 system. 

None of this belongs in the book.
A small revision of the sentence in question might be:

> "It was eventually developed into a rigorous formal system in its own right under the name “typed λ-calculus”, see Church [Chu40, Chu41] for an early formulation."



Steve

**Comment by vladimirias** [comment]
> On Feb 28, 2015, at 2:23 PM, Steve Awodey <notifications@github.com> wrote:
> 
> > "It was eventually developed into a rigorous formal system in its own right under the name “typed λ-calculus”, see Church [Chu40, Chu41] for an early formulation.”

This sentence is wrong. It was not developed into a rigorous formal system under this name. This name appeared much later than the first rigorous formulations of type theory appeared.

Vladimir.

**Comment by awodey** [comment]
well, perhaps ambiguous.  
So let’s take this instead:

	It was eventually developed into a rigorous formal system in its own right called “typed λ-calculus”, 
	see Church [Chu40, Chu41] for an early formulation.

Steve

**Comment by pthariensflame** [comment]
@awodey How about "*now* called", instead of just "called"?

**Comment by vladimirias** [comment]
“typed lambda calculus” is not a type sytem. It is a family of type systems, one for each set of generating types.

The type system devised by Church in Chu40 on the other hand is a type system.

Vladimir.

**Comment by awodey** [comment]
who said anything about "type systems"?

**Comment by awodey** [comment]
> On Feb 28, 2015, at 1:31 PM, Vladimir Voevodsky <notifications@github.com> wrote:
> 
> This
> 
> > The idea of homotopy type theory arose around 2006 in independent work by Awodey and Warren [AW09] and Voevodsky [Voe06], but it was inspired by Hofmann and Streicher’s earlier groupoid interpretation [HS98].
> > 
> 
> is material error. The idea of generalizing the groupoid interpretation of Hofmann and Streicher to an interpretation in Kan complexes or weak factorization systems or Quillen model categories was widely discussed in the Fall of 2006. Among those discussing it there were Streicher himself, Richard Garner and Michael Warren. The notes written at about that time by Garner and Warren are available online.
> 
> Vladimir.
> 
> 

you have the history wrong.
the meeting and discussions in Uppsala in the fall of 2006 were organized AFTER the work of Awodey and Warren was first presented in public at a meeting in the summer of 2006.  The ideas were also presented to Per Martin-Löf and Andre’ Joyal in an informal seminar in Montreal in the Summer of 2006.  
you were not there — I suggest you rely on the advice and testimony of those who were, whether Awodey and Warren deserve credit.

**Comment by mikeshulman** [comment]
> “typed lambda calculus” is not a type sytem. It is a family of type systems, one for each set of generating types.

Whether or not that's true, I think it's splitting far too many hairs for the introduction to the book.  Plus, Steve's sentence says "formal system" not "type system" anyway.

**Comment by awodey** [comment]
this has degenerated into hair-splitting, distortions, and outright falsehoods.

**Comment by awodey** [comment]
According to what I have learned asking many people Michael Warren is the person who was the first to discover the connection between the J-eleiminator and the Bousfield-Quillen axioms.

Richard Garner described how he came up with similar (but different) ideas, probably a few months later. His notes (http://comp.mq.edu.au/~rgarner/Papers/Uppsala.pdf <http://comp.mq.edu.au/~rgarner/Papers/Uppsala.pdf>) clearly confirm that he worked independently.

Everybody I have spoken to said that your contribution to the discovery of these ideas was minimal. This is confirmed by Warren’s thesis prospectus (http://mawarren.net/papers/prospectus.pdf <http://mawarren.net/papers/prospectus.pdf>). The name "Awodey” is mentioned there exactly once (on p.4) in connection with a proposition describing the specialization of the results to the case of the extensional type theory.

Vladimir.

**Comment by awodey** [comment]
I don’t know who your sources are, but they are not accurate.  Michael Warren was my student and we were working together.
This was always presented as joint work, and anyone who was present at the early talks will confim this.
Perhaps someone will volunteer a confirmation on this list, but I don’t really think it is appropriate to have this sort of discussion here.
I only reply now because I cannot allow you to continue to spread such falsehoods in public without any rebuttal.
My contributions are a fact beyond doubt, known to many people who were involved in the early development, and I think that your attempts to dispute them are a dishonest and unethical abuse of your influence which should not be tolerated.

Steve

**Comment by andrejbauer** [comment]
This pull request has detirorated into a discussion which does not belong here. If there is agreement on a minimal change that would fix the errors, then let us do that, but please let us stick to making only minor changes that fix factual errors. It looks to me that we have:

1. A proposal to change the statement about what was going on in 1930's.
2. A proposal to change the statement about what was going on in 2006.
3. Minor technical improvements.

It may be wiser to close this issue and discuss the three separately. The second issue is contentious and should be discussed privately first. Opinions?

**Comment by vladimirias** [comment]
> On Mar 1, 2015, at 6:08 PM, Andrej Bauer <notifications@github.com> wrote:
> 
> This pull request has detirorated into a discussion which does not belong here. If there is agreement on a minimal change that would fix the errors, then let us do that, but please let us stick to making only minor changes that fix factual errors. It looks to me that we have:
> 
> 	• A proposal to change the statement about what was going on in 1930's.
> 	• A proposal to change the statement about what was going on in 2006.
> 	• Minor technical improvements.
> It may be wiser to close this issue and discuss the three separately. The second issue is contentious and should be discussed privately first. Opinions?
> 
> —
> Reply to this email directly or view it on GitHub.
> 

1. The first issue is not about what was going on in 1930 but about being precise in stating things. Saying that type theory of Russell was made into a rigorous formal system under the name of lambda-calculus is mixing apples and oranges.

2. The second proposal is simply to remove such contentious material from the book.

3. The rest of the corrections are also about being precise. For example:

"Voevodsky recognized that the simplicial interpretation of type theory satisfies a further crucial property, dubbed \emph{univalence}, "

is incorrect. There is no "the simplicial interpretation" that was shown to satisfy a property. An interpretation was constructed that satisfied the property.

Vladimir.

**Comment by andrejbauer** [comment]
I'm just saying it would be easier to split these and treat them separately because it looks like we can agree on two out of three (first and certainly third), but there is no way to accept only two thirds of a pull request.

**Comment by peterlefanulumsdaine** [comment]
I agree completely with Andrej’s suggestion of splitting it up.  Any intended correction is about “being precise”, but these three parts are on very different topics, some clear-cut, some much less so.

I also second his suggestion that the more contentious parts would be better discussed privately, to try to reach a consensus before proposing rewrites (and also agree with Mike S that major changes should be left for a second edition).

**Comment by mikeshulman** [comment]
This PR has actually already been closed, so yes, let's open separate ones for each suggestion.

**Comment by andrejbauer** [comment]
I know this issue is closed and that I myself proposed that issue 2 be discussed privately, but since there has been a public discussion, I would like to put on record my own testimony. In April 2007 I was at CMU. Steve Awodey and Michael Warren were excited about a new connection between identity types and model categories. This was very good news as Steve and I previously tried to make sense of the intensional identity types using various approaches, but failed to do so. I spoke at length with Michael who explained the technicalities to me -- the meeting was probably set up by Steve because he wanted Michael and me to meet and I did not know Michael well at the time. It's even possible this was the first time I met Michael. I cannot remember explicit words, but from that visit I definitely took away the memory that the result was joint work of Steve and Michael. Never until recently have I heard any claims to the contrary.

**Pull request closed without merging**